src/routes/user.js

```diff
@@ -29,4 +36,14 @@ router.get('/settings', requireSessionUser, settingsPage)
 // sign the user out
 router.get('/logout', asyncMiddleware(logout))
 
+// unsubscribe from emails
+router.get('/unsubscribe', urlEncodedParser, asyncMiddleware(unsubscribePage))
+router.post('/unsubscribe', asyncMiddleware(unsubscribeFromEmails))
+
+router.get(
+  '/unsubscribe-monthly',
+  urlEncodedParser,
+  asyncMiddleware(unsubscribeMonthlyPage)
+)
```

Comment:
Directly unsubscribe users that follow the “unsubscribe” link from the monthly emails.